config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -52,27 +52,30 @@ periodics:
         requests:
           cpu: 4
           memory: "8Gi"
-
   annotations:
     testgrid-dashboards: sig-release-master-blocking, google-unit
     testgrid-tab-name: build-master-fast
     testgrid-alert-email: kubernetes-release-team@googlegroups.com
     description: 'Ends up running: make quick-release'
+
 - interval: 4h
-  name: ci-kubernetes-build-debian-unstable
+  name: ci-release-build-packages-debs
+  decorate: true
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: release
+    base_ref: master
+    path_alias: "k8s.io/release"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bootstrap:v20190703-1f4d616
-      args:
-      - --repo=k8s.io/release
-      - --root=/go/src
-      - --timeout=180
-      - --scenario=execute
-      - --
-      - ./debian/jenkins.sh
+    - image: quay.io/k8s/release-tools:latest
```

Comment:
Not sure what this is built from, but FWIW I would recommend a followup PR to add a `release-tools` dir to https://github.com/kubernetes/test-infra/tree/master/images and go back to using gcr.io

Reply:
@spiffxp -- I was planning on adding them to k/release. Is there an advantage to having them in k/test-infra instead?

Reply:
we auto-build and push images that live in that dir to gcr.io via https://github.com/kubernetes/test-infra/tree/master/images/builder, tagged with the date and sha they were built, it's a pattern that's worked pretty well to know exactly what's being used